plans/fix-1899-splitter-pointer-capture.md

```
@@ -0,0 +1,107 @@
+| 修正後 | `DIV`（セパレータが追いついた） | 505 | **505** | 追随 |
+
+離した後の挙動:
+
+| | リリース後にボタンを離したまま左へ動かす |
+|---|---|
+| 修正前 | 幅が **360 → 535.5 → 669 → 709** と変わり続ける（ドラッグが生きている） |
+| 修正後 | **360 のまま**（終了している） |
+
+3 本すべての分割バーを実機で駆動（切り出し後のビルド）:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)